src/Templates/src/templates/maui-multihead/MauiApp.1/MauiApp.1.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>DOTNET_TFM</TargetFramework>
+		<SingleProject>true</SingleProject>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<UseMaui>true</UseMaui>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<!-- App Icon -->
```

Comment:
MauiIcon and MauiSplashScreen are commented out, but the Resources/Splash is still included. I think we should remove them altogether for this template, esp since the primary usage is for upgrades and they will have separate implementations.